text/0000-prelude-additions.md

```diff
@@ -0,0 +1,56 @@
+- Feature Name: NA
+- Start Date: 2015-04-03
+- RFC PR: (leave this empty)
+- Rust Issue: (leave this empty)
+
+# Summary
+
+Add `Default`, `IntoIterator` and `ToOwned` trait to the prelude.
+
+# Motivation
+
+Each trait has a distinct motivation:
+
+* For `Default`, the ergonomics have vastly improved now that you can
+  write `MyType::default()` (thanks to UFCS). Thanks to this
+  improvement, it now makes more sense to promote widespread use of
+  the trait.
+
+* For `IntoIterator`, promoting to the prelude will make it feasible
+  to deprecate the inherent `into_iter` methods and directly-exported
+  iterator types, in favor of the trait (which is currently redundant).
+
+* For `ToOwned`, promoting to the prelude would add a uniform,
+  idiomatic way to acquire an owned copy of data (including going from
+  `str` to `String`, for which `Clone` does not work).
+
+# Detailed design
+
+* Add `Default`, `IntoIterator` and `ToOwned` trait to the prelude.
+
+* Deprecate inherent `into_iter` methods.
```

Comment:
We may want to consider removing these methods entirely actually as it's difficult to call a trait method instead of an inherent method sometimes.